Projects/Services/APIKit/Sources/StudyRoomsAPI.swift

```diff
@@ -64,6 +64,13 @@ extension StudyRoomsAPI: DmsAPI {
 
     public var task: Moya.Task {
         switch self {
+        case let .fetchSeatTypes(studyroomID):
+            return .requestParameters(
+                parameters: [
+                    "study_room_id": studyroomID
+                ],
+                encoding: URLEncoding.queryString
+            )
         case let .fetchStudyRoomList(timeSlot):
```

Comment:
```suggestion
            )

        case let .fetchStudyRoomList(timeSlot):
```
한 줄 줄바꿈 스타일 가이드 맞춰야할거같아요!

Reply:
죄송해요 ㅠㅠ 맨날 안 바꾸는 것 같네요 앞으로 한 번씩 더 확인하겠습니다!